locales-ar.xml

```diff
@@ -70,6 +70,12 @@
 
     <!-- ORDINALS -->
     <term name="ordinal"></term>
+    <term name="ordinal-01"></term>
+    <term name="ordinal-02"></term>
+    <term name="ordinal-03"></term>
+    <term name="ordinal-11"></term>
+    <term name="ordinal-12"></term>
+    <term name="ordinal-13"></term>
```

Comment:
@abdealikhurrum, we'd be happy to accept the translation for the "available at" term, but these empty `<term name="ordinal-xx"></term>` terms are unnecessary and shouldn't make a difference.